mail/urlscan/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        firecat53 urlscan 0.9.5
+categories          mail python
+platforms           darwin
+license             GPL-2
+maintainers         {@ryanakca debian.org:rak} \
+                    openmaintainer
+description         extract and browse the URLs in an email
+long_description    ${name} lets you easily browse URLs contained in an email\
+                    and launch a web browser to view them. It is a replacement\
+                    for the "urlview" program and integrates with the mutt MUA.
+
+github.tarball_from archive
+
+python.default_version     39
```

Comment:
this doesn't follow the 4-spaces; for this line you can either do: ```python.default_version 39``` or have the argument at multiples of 4 spaces: ```python.default_version  39```